### PR TITLE
update-modules: remove kirkstone warnings

### PR DESCRIPTION
**Pull request by TheYoctoJester**

### CODE REVIEW

File: meta-mender-update-modules/conf/layer.conf [hunk 1]
```diff
@@ -11,4 +11,4 @@ BBFILE_COLLECTIONS += "mender-update-modules"
 BBFILE_PATTERN_mender-update-modules = "^${LAYERDIR}/"
 BBFILE_PRIORITY_mender-update-modules = "15"
 
-LAYERSERIES_COMPAT_mender-update-modules = "kirkstone"
+LAYERSERIES_COMPAT_mender-update-modules = "kirkstone"
```
Comment (kacf) [inline meta-mender-update-modules/conf/layer.conf:14]: I guess it's fine, but what was the warning? That the file ends in a newline? Isn't that what it *should* end with?

**Review by kacf — APPROVED** [review]

File: meta-mender-update-modules/conf/layer.conf (hunk 1 above)
Comment (oleorhagen) [reply meta-mender-update-modules/conf/layer.conf:14]: It's the opposice, no? THat it does _not_ end in a newline?

Comment (TheYoctoJester) [reply meta-mender-update-modules/conf/layer.conf:14]: Good catch indeed, yet, where did you get a warning?

Comment (kacf) [reply meta-mender-update-modules/conf/layer.conf:14]: I'm very confused now. Removing the newline was the only change in this PR...

**Pull request merged**